CIFAR10/train/fastai_pytorch.json

```diff
@@ -0,0 +1,11 @@
+{
+    "version": "v1.0",
+    "author": "fast.ai + students",
+    "authorEmail": "j@fast.ai",
+    "framework": "fastai / pytorch",
+    "codeURL": "https://github.com/fastai/imagenet-fast/",
+    "model": "Custom Wide Resnet",
+    "hardware": "8 * V100 (AWS p3.xlarge)",
+    "costPerHour": 8.68,
```

Comment:
We do not allow spot pricing. Please update the cost to reserved instance cost.

Reply:
updated